src/db_window.rs

```diff
@@ -61,14 +63,12 @@ pub fn process_blob(
         let r_blob = blob.read().unwrap();
         (r_blob.slot(), r_blob.index())
     };
-    let leader = leader_scheduler.read().unwrap().get_leader_for_slot(slot);
+    let bank = bank_forks.read().unwrap().working_bank();
+    let epoch = slot / bank.slots_per_epoch();
+    let _leader = LeaderScheduler1::default().slot_leader_by(&bank, |_, _, _| (slot, epoch));
```

Comment:
Why bother?

Reply:
I left it in there due to this comment in the next line:

```
    // TODO: Once the original leader signature is added to the blob, make sure that
    // the blob was originally generated by the expected leader for this slot
```

Reply:
okay cool, good stuff